src/Accounts/Accounts/Account/ConnectAzureRmAccount.cs

```diff
@@ -404,5 +410,35 @@ public void OnImport()
             }
 #endif
         }
+
+        private void DisableAutosave(IAzureSession session, bool writeAutoSaveFile, out ContextAutosaveSettings result)
+        {
+            result = new ContextAutosaveSettings
+            {
+                Mode = ContextSaveMode.Process
+            };
+
+            session.ARMContextSaveMode = ContextSaveMode.Process;
+            var memoryCache = session.TokenCache as AuthenticationStoreTokenCache;
+            if (memoryCache == null)
+            {
+                var diskCache = session.TokenCache as ProtectedFileTokenCache;
+                memoryCache = new AuthenticationStoreTokenCache(new AzureTokenCache());
+                if (diskCache != null && diskCache.Count > 0)
+                {
+                    memoryCache.Deserialize(diskCache.Serialize());
+                }
+
+                session.TokenCache = memoryCache;
+            }
+
+            if (writeAutoSaveFile)
+            {
+                FileUtilities.DataStore = session.DataStore;
+                FileUtilities.EnsureDirectoryExists(session.ProfileDirectory);
+                string autoSavePath = Path.Combine(session.ProfileDirectory, ContextAutosaveSettings.AutoSaveSettingsFile);
+                session.DataStore.WriteFile(autoSavePath, JsonConvert.SerializeObject(result));
+            }
```

Comment:
`writeAutoSaveFile` is allways false. We can remove this variable.

Reply:
Done